service/models/product.py

```diff
@@ -11,18 +11,117 @@
 
 """
 
-class Product():
-  def create():
-    pass
+from flask import Flask
+from .model_utils import db,logger,Availability,DataValidationError
 
-  def read():
-    pass
+class Product(db.Model):
+
+  __tablename__= 'product'
+
+  app:Flask = None
+
+  id = db.Column(db.Integer,primary_key = True)
+  name = db.Column(db.String(64), nullable=False)
+  price = db.Column(db.DECIMAL,nullable=False)
+  status = db.Column(db.Enum(Availability),nullable=False,default=1)
+  pic_url = db.Column(db.Text,nullable=True)
+  short_desc = db.Column(db.Text,nullable=True)
+  
```

Comment:
Do we still want `category` as an attribute? It was mentioned in the ticket, but if you think it's not needed, I'm happy to let it go.